build_tools/circle/build_push_doc.sh

```diff
@@ -13,7 +13,7 @@ fi
 
 # The version is retrieved from the CIRCLE_TAG. If there is no version, we just
 # call it 0.0.0, since we won't be pushing anyways (not master and no tag)
-if [[ ! -z ${CIRCLE_TAG} ]]; then
+if [[ ! -z ${CIRCLE_TAG} && ${CIRCLE_TAG} =~ '^v?[0-9]+\.[0-9]+\.?[0-9]*?[a-zA-Z]+[0-9]*$' ]]; then
```

Comment:
@tgsmith61591 Is this okay? This way, it _should_ skip doc deployments on non-prod tags. You know this step better than me, so I want to confirm it looks okay

Reply:
Disregard. I moved this check [here](https://github.com/alkaline-ml/pmdarima/pull/283/files#diff-edb3eabb049d60b13c2b45c3e5ece16fR163)